packages/graphql/src/schema-model/entity/model-adapters/ConcreteEntityAdapter.ts

```diff
@@ -159,6 +159,24 @@ export class ConcreteEntityAdapter {
         return Array.from(this.attributes.values()).filter((attribute) => attribute.isTemporal());
     }
 
+    public getRelayId(): AttributeAdapter | undefined {
+        // TODO: make this O(1) or memoize
+        for (const attr of this.attributes.values()) {
+            if (attr.annotations.relayId) {
+                return attr;
+            }
+        }
+        return undefined;
+    }
```

Comment:
You have access to `globalIdField`, which was introduced by the schema generator changes
```suggestion
```